### PR TITLE
Fix cuda memory leak

### PR DESCRIPTION
**Pull request by FrostML**
### PR types

Bug fixes

### PR changes

APIs

### Description

Fix cuda memory leak caused by cublas handle.

### CODE REVIEW

**Comment by ZeyuChen** [comment]
后续FT的提测得让QA补充上这块的验证。

File: paddlenlp/ops/faster_transformer/src/fusion_decoding_op.cu [hunk 1]
```diff
@@ -282,6 +282,7 @@ std::vector<paddle::Tensor> decoding_kernel(
         "Faster Transformer. ");
   }
   delete[] params;
+  cublasDestroy(cublas_handle_);
```
Comment (guoshengCS) [inline paddlenlp/ops/faster_transformer/src/fusion_decoding_op.cu]: cublasCreate在DecodingCUDAForward里面的话，这个放到和cublasCreate一样放到DecodingCUDAForward里面吧

File: paddlenlp/ops/faster_transformer/src/fusion_gpt_op.cu [hunk 2]
```diff
@@ -147,6 +147,7 @@ std::vector<paddle::Tensor> gpt2_kernel(
 
   delete gpt2_decoding;
   delete[] params;
+  cublasDestroy(cublas_handle_);
```
Comment (guoshengCS) [inline paddlenlp/ops/faster_transformer/src/fusion_gpt_op.cu]: 同上

File: paddlenlp/ops/faster_transformer/src/fusion_decoding_op.cu (hunk 1 above)
Comment (FrostML) [reply paddlenlp/ops/faster_transformer/src/fusion_decoding_op.cu]: Done. Thanks.

File: paddlenlp/ops/faster_transformer/src/fusion_gpt_op.cu (hunk 2 above)
Comment (FrostML) [reply paddlenlp/ops/faster_transformer/src/fusion_gpt_op.cu]: Done. Thanks.

**Review by guoshengCS — APPROVED** [review]

**Pull request merged**